server/test/unit/com/thoughtworks/go/server/materials/MaterialUpdateServiceTest.java

```diff
@@ -188,6 +205,11 @@ public void shouldReturnImplementerOfSvnPostCommitHookAndPerformMaterialUpdate_W
         spyService.notifyMaterialsForUpdate(username, params, result);
         verify(svnPostCommitHookImplementer).prune(anySet(), eq(params));
         verify(spyService).updateMaterial(svnMaterial);
+
+        HttpLocalizedOperationResult acceptedResult = new HttpLocalizedOperationResult();
+        acceptedResult.accepted(LocalizedMessage.string("MATERIAL_SCHEDULE_NOTIFICATION_ACCEPTED"));
+
+        assertThat(result, is(acceptedResult));
```

Comment:
The equals was not implemented earlier. We have to do that for checking the localizedmessage. Will change it consistent at other places and synchronize this.